happi/loader.py

```diff
@@ -177,6 +177,14 @@ def create_arg(arg):
     args = [create_arg(arg) for arg in item.args]
     kwargs = dict((key, create_arg(val))
                   for key, val in item.kwargs.items())
+    # maybe filter out null kwargs
+    if not item._info_attrs['kwargs'].include_default_as_kwarg:
```

Comment:
The way it's implemented, either all kwargs will be subject to filtering, or none of them will.  This property gets set on the kwargs EntryInfo, rather than individually for each EntryInfo that might get pulled into a kwarg.  

Am I this question right?

Reply:
Got it, I was confused because of the seeming (near) contradiction between the above implementation and the desire to not do something special with the kwargs `EntryInfo`. I realize that you need to redefine `kwargs` anyway in these subclasses so the distinction is pretty minor.